executor/join.go

```diff
@@ -419,6 +420,7 @@ type NestedLoopJoinExec struct {
 	BigFilter   expression.Expression
 	OtherFilter expression.Expression
 	schema      *expression.Schema
+	outer       bool
```

Comment:
Oh, there is TODO on `doJoin`, but why not add it when you actually handle outer join case?